tests/appveyor/mssql.appveyor.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.8/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="../../vendor/autoload.php"
+>
+    <php>
+        <ini name="error_reporting" value="-1" />
+
+        <var name="db_type" value="pdo_sqlvr"/>
```

Comment:
Considering the config XML files live in a CI service specific option can we drop the service name from the XML files?
so change `<db_name>.<db_version>.<dbal_driver_name>.appveyor.xml` to `<db_name>.<db_version>.<dbal_driver_name>.xml` ?